images/router/haproxy/conf/haproxy-config.template

```diff
@@ -22,6 +22,9 @@
 {{/* cidrListPattern: Match a space separated list of CIDRs; e.g. 192.168.21.23/24 192.10.2.12 */}}
 {{- $cidrListPattern := printf `(?:%s(?: +%s)*)` $cidrPattern $cidrPattern -}}
 
+{{/* cookie name pattern: */}}
+{{- $cookieNameRE := "[a-zA-Z0-9_-]+" -}}
```

Comment:
Can it start with a '-' or '_' ?
Is it to follow the TEXT as defined in https://tools.ietf.org/html/rfc2616#section-2.2

Reply:
@rajatchopra  It supports leading '-' and '_'.  It is a subset of https://tools.ietf.org/html/rfc2616#section-2.2. In particular embedded <sp>, <tab> and punctuation (other than '-' and ._.) are excluded.
This is as far as @knobunc wanted to go.

Reply:
@rajatchopra we can allow more if you want.  But I couldn't work out how to find out what was safe in the haproxy file at that point... or how to escape things.

@pecameron I realized the other regexes are named *Pattern not *RE... can we change this to match please.

Reply:
Also... the comment in the commit message does not match the new regex...

Reply:
And, the pattern should be surrounded by ``s not ""s